src/java/org/apache/cassandra/db/ExpirationDateOverflowHandling.java

```diff
@@ -62,23 +62,23 @@
     }
 
     public static final String MAXIMUM_EXPIRATION_DATE_EXCEEDED_WARNING = "Request on table {}.{} with {}ttl of {} seconds exceeds maximum supported expiration " +
-                                                                          "date of 2038-01-19T03:14:06+00:00 and will have its expiration capped to that date. " +
+                                                                          "date of 2106-02-07T06:28:14+00:00 and will have its expiration capped to that date. " +
```

Comment:
Same as before, the current max expiration time is `2106-02-07T06:28:13+00:00`, due to `MAX_DELETION_TIME` being `CassandraUInt.maxValueAsLong() - 2` instead of `CassandraUInt.maxValueAsLong() - 1`.

Reply:
All these batched in the original comment on this subject